core/src/OpenACC/Kokkos_OpenACC.hpp

```diff
@@ -89,6 +90,8 @@ class OpenACC {
 
   OpenACC();
 
+  OpenACC(int async_arg);
+
```

Comment:
I would prefer an `explicit` on the one argument constructor, users might fuck up some things otherwise.
If we give the passed parameter a default value we could get rid of the default ctor

Reply:
Done